{"uid":"e0d608b8c81bcacc","name":"Create a new Order for existing Customer","fullName":"TestOrder.TestOrder#test_create_order_customer_5___frugal_latido_com_np____Test_123____9813000000____female____Luigi__male_____Mahogany_Antique____41____Golden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_","historyId":"3323c969f29de0927878d0c7ee541282","time":{"start":1687502358411,"stop":1687502428590,"duration":70179},"description":"Create an Order for existing Customer ","descriptionHtml":"<p>Create an Order for existing Customer</p>\n","status":"broken","statusMessage":"AttributeError: 'TestOrder' object has no attribute 'log'","statusTrace":"self = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9813000000', gender = 'female', productname = 'Luigi (male)'\nleatherProfile = 'Mahogany Antique', leatherSize = '41'\nhardware = 'Golden Antique', lining = 'Black', polyfill = 'Yes', size = '40'\narmhole = 5.6, height = 4.4, shoulder = 5, weight = 46, length = 8, arms = 12\nhips = 11, chest = 7, waist = 1, sleeves = 9, bodytype = 'Comfort'\nremark = 'Test 004', date = '2023-06-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for existing Customer\")\n    @allure.description(\"Create an Order for existing Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forCustomer\"))\n    @unpack\n    def test_create_order_customer(self, username, password, contactNo, gender, productname, leatherProfile,\n                                   leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(username, password)\n        test_case_name = f\"Create a new Order for existing Customer : {contactNo} \"\n        try:\n>           self.assertEqual(\n                self.Order.createOrder_Customer(contactNo, gender, productname, leatherProfile, leatherSize, hardware,\n                                                lining, polyfill, size, armhole, height, shoulder, weight, length, arms,\n                                                hips, chest, waist, sleeves, bodytype, remark, date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n\nTestOrder.py:43: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nfirst = None, second = 'Order Succesfully Created'\nmsg = 'Verification Failed to Create a new Order.'\n\n    def _patched_equals(self, first, second, msg=None):\n        try:\n>           old(self, first, second, msg)\n\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nfirst = None, second = 'Order Succesfully Created'\nmsg = 'Verification Failed to Create a new Order.'\n\n    def assertEqual(self, first, second, msg=None):\n        \"\"\"Fail if the two objects are unequal as determined by the '=='\n           operator.\n        \"\"\"\n        assertion_func = self._getAssertEqualityFunc(first, second)\n>       assertion_func(first, second, msg=msg)\n\nC:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python310\\lib\\unittest\\case.py:845: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nfirst = None, second = 'Order Succesfully Created'\nmsg = \"None != 'Order Succesfully Created' : Verification Failed to Create a new Order.\"\n\n    def _baseAssertEqual(self, first, second, msg=None):\n        \"\"\"The default assertEqual implementation, not type specific.\"\"\"\n        if not first == second:\n            standardMsg = '%s != %s' % _common_shorten_repr(first, second)\n            msg = self._formatMessage(msg, standardMsg)\n>           raise self.failureException(msg)\nE           AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.\n\nC:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python310\\lib\\unittest\\case.py:838: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9813000000', gender = 'female', productname = 'Luigi (male)'\nleatherProfile = 'Mahogany Antique', leatherSize = '41'\nhardware = 'Golden Antique', lining = 'Black', polyfill = 'Yes', size = '40'\narmhole = 5.6, height = 4.4, shoulder = 5, weight = 46, length = 8, arms = 12\nhips = 11, chest = 7, waist = 1, sleeves = 9, bodytype = 'Comfort'\nremark = 'Test 004', date = '2023-06-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for existing Customer\")\n    @allure.description(\"Create an Order for existing Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forCustomer\"))\n    @unpack\n    def test_create_order_customer(self, username, password, contactNo, gender, productname, leatherProfile,\n                                   leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(username, password)\n        test_case_name = f\"Create a new Order for existing Customer : {contactNo} \"\n        try:\n            self.assertEqual(\n                self.Order.createOrder_Customer(contactNo, gender, productname, leatherProfile, leatherSize, hardware,\n                                                lining, polyfill, size, armhole, height, shoulder, weight, length, arms,\n                                                hips, chest, waist, sleeves, bodytype, remark, date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n    \n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException, AttributeError) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\",\n                          attachment_type=AttachmentType.PNG)\n>           self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\nE           AttributeError: 'TestOrder' object has no attribute 'log'\n\nTestOrder.py:53: AttributeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestOrder","time":{"start":1687500628072,"stop":1687500628072,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"testStage":{"description":"Create an Order for existing Customer ","status":"broken","statusMessage":"AttributeError: 'TestOrder' object has no attribute 'log'","statusTrace":"self = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9813000000', gender = 'female', productname = 'Luigi (male)'\nleatherProfile = 'Mahogany Antique', leatherSize = '41'\nhardware = 'Golden Antique', lining = 'Black', polyfill = 'Yes', size = '40'\narmhole = 5.6, height = 4.4, shoulder = 5, weight = 46, length = 8, arms = 12\nhips = 11, chest = 7, waist = 1, sleeves = 9, bodytype = 'Comfort'\nremark = 'Test 004', date = '2023-06-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for existing Customer\")\n    @allure.description(\"Create an Order for existing Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forCustomer\"))\n    @unpack\n    def test_create_order_customer(self, username, password, contactNo, gender, productname, leatherProfile,\n                                   leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(username, password)\n        test_case_name = f\"Create a new Order for existing Customer : {contactNo} \"\n        try:\n>           self.assertEqual(\n                self.Order.createOrder_Customer(contactNo, gender, productname, leatherProfile, leatherSize, hardware,\n                                                lining, polyfill, size, armhole, height, shoulder, weight, length, arms,\n                                                hips, chest, waist, sleeves, bodytype, remark, date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n\nTestOrder.py:43: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nfirst = None, second = 'Order Succesfully Created'\nmsg = 'Verification Failed to Create a new Order.'\n\n    def _patched_equals(self, first, second, msg=None):\n        try:\n>           old(self, first, second, msg)\n\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nfirst = None, second = 'Order Succesfully Created'\nmsg = 'Verification Failed to Create a new Order.'\n\n    def assertEqual(self, first, second, msg=None):\n        \"\"\"Fail if the two objects are unequal as determined by the '=='\n           operator.\n        \"\"\"\n        assertion_func = self._getAssertEqualityFunc(first, second)\n>       assertion_func(first, second, msg=msg)\n\nC:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python310\\lib\\unittest\\case.py:845: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nfirst = None, second = 'Order Succesfully Created'\nmsg = \"None != 'Order Succesfully Created' : Verification Failed to Create a new Order.\"\n\n    def _baseAssertEqual(self, first, second, msg=None):\n        \"\"\"The default assertEqual implementation, not type specific.\"\"\"\n        if not first == second:\n            standardMsg = '%s != %s' % _common_shorten_repr(first, second)\n            msg = self._formatMessage(msg, standardMsg)\n>           raise self.failureException(msg)\nE           AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.\n\nC:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python310\\lib\\unittest\\case.py:838: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <TestsCases.TestOrder.TestOrder testMethod=test_create_order_customer_5___frugal_latido_com_np____Test_123____98130000...lden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Comfort____Test_004____2023_06_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9813000000', gender = 'female', productname = 'Luigi (male)'\nleatherProfile = 'Mahogany Antique', leatherSize = '41'\nhardware = 'Golden Antique', lining = 'Black', polyfill = 'Yes', size = '40'\narmhole = 5.6, height = 4.4, shoulder = 5, weight = 46, length = 8, arms = 12\nhips = 11, chest = 7, waist = 1, sleeves = 9, bodytype = 'Comfort'\nremark = 'Test 004', date = '2023-06-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for existing Customer\")\n    @allure.description(\"Create an Order for existing Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forCustomer\"))\n    @unpack\n    def test_create_order_customer(self, username, password, contactNo, gender, productname, leatherProfile,\n                                   leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(username, password)\n        test_case_name = f\"Create a new Order for existing Customer : {contactNo} \"\n        try:\n            self.assertEqual(\n                self.Order.createOrder_Customer(contactNo, gender, productname, leatherProfile, leatherSize, hardware,\n                                                lining, polyfill, size, armhole, height, shoulder, weight, length, arms,\n                                                hips, chest, waist, sleeves, bodytype, remark, date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n    \n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException, AttributeError) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Failed_for_customer\",\n                          attachment_type=AttachmentType.PNG)\n>           self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\nE           AttributeError: 'TestOrder' object has no attribute 'log'\n\nTestOrder.py:53: AttributeError","steps":[],"attachments":[{"uid":"429d10a8f004a0f8","name":"Failed_for_customer","source":"429d10a8f004a0f8.png","type":"image/png","size":125286},{"uid":"7e402d4d5f14268a","name":"log","source":"7e402d4d5f14268a.txt","type":"text/plain","size":3313}],"parameters":[],"attachmentsCount":2,"shouldDisplayMessage":true,"stepsCount":0,"hasContent":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestOrder::0","time":{"start":1687502716177,"stop":1687502716177,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"labels":[{"name":"feature","value":"Order List"},{"name":"severity","value":"normal"},{"name":"suite","value":"TestOrder"},{"name":"subSuite","value":"TestOrder"},{"name":"host","value":"0day"},{"name":"thread","value":"2692-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"TestOrder"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":3,"unknown":0,"total":4},"items":[{"uid":"9657c7338c48779c","status":"passed","time":{"start":1686891778847,"stop":1686891839433,"duration":60586}},{"uid":"696d4cd51ada06ab","status":"passed","time":{"start":1686288835352,"stop":1686288901865,"duration":66513}},{"uid":"ea2891b883eb9a92","status":"passed","time":{"start":1685710787685,"stop":1685710843636,"duration":55951}}]},"tags":[]},"source":"e0d608b8c81bcacc.json","parameterValues":[]}